bin/refname-callback.py

```diff
@@ -0,0 +1,46 @@
+def convertRef(refname):
+    from packaging import version
+
+    last_tag_name = b'${LAST_TAG_IN_SINGLEREPO}'
+    tag_prefix = b'${TAG_PREFIX}'
+
+    last_tag_name = b'2.7.0'
+    tag_prefix = b'prefix/'
+
+## ------- cut here ✂ --------
```

Comment:
```suggestion
following code is used in split-repo.sh. This file just demonstrates and tests effect of the code used in split-repo.sh
## ------- cut here ✂ --------
```

Reply:
Akorát by to musel být comment, upravím ;)

Reply:
jaj, sorry :D